api/machine/machine.proto

```diff
@@ -73,16 +73,33 @@ service MachineService {
 // ApplyConfiguration describes a request to assert a new configuration upon a
 // node.
 message ApplyConfigurationRequest {
+  enum Mode {
+    AUTO = 0;
```

Comment:
should the default be `REBOOT = 0` so that we have backwards compatibility with clients? otherwise how do we know that `talosctl apply-config` from v0.14 is using?